### ISSUE TITLE
No hd button

### ISSUE THREAD

**Issue by KakarN** [Hussnain1/Video.js-HD-Toggle-Plugin#3]
Hello,
I have added the hd.js and hd.css (from http://jsfiddle.net/Hussnain1/unrjrqee/) inside my htm filel, but there's no hd button inside my video player. What may be the problem? Please help me solve this.

Thank you.

**Comment by bmatthewshea** [comment]
Yes, just tried it under current / 6.2.7 .. nope: http://jsfiddle.net/aw992cmk/
(4.6.4 is loaded on the fiddle page that works . .)
